lib/cli/ares.js

```diff
@@ -160,3 +160,31 @@ export async function uninstallApp(appId, deviceName) {
   log.info(`Uninstalling app '${appId}'`);
   await runDeviceCmd(ARES_INSTALL, deviceName, ['-r', appId]);
 }
+
+
+/**
+ * Return the list of running apps
+ * @param {*} deviceName
+ */
+export async function listRunningApps(deviceName) {
+  //
+  //
+  //com.suitest.watchme.app
```

Comment:
I don't think we need these comments do we?

Reply:
ooh, these changes are not related. my local past some changes...